front-end/src/components/modal/create/GraphSettings.js

```diff
@@ -27,20 +34,21 @@ const GraphSettings = ({ movePage, setGraphType, setLiveCheck }) => {
         <div className="graphTypesBox">
           <div className="graphOptionsText">Graph Types</div>
           <div className="GraphTypeSelect">
-            <select id="graphTypeSelect" >
+            <select id="graphTypeSelect" onChange={(e) => { handleTypeSelect(e) }}>
               <option value="line">Line</option>
               <option value="spline">Spline</option>
               <option value="scatter">Scatter</option>
               <option value="coloredline">XYColour</option>
+              <option value="video">Video</option>
```

Comment:
I think put video higher on the drop down since things like scatter and spline are never used and kind of irrelevant while video is useful.

Reply:
moved to 2nd position